### ISSUE TITLE
Consistent Names for Feature Spec

### ISSUE THREAD

**Issue by ChaelCodes** [ChaelCodes/ConfBuddies#34]
# Description
Our feature tests have very non-descriptive names in the `describe`. They're named after the object from the route. So it'll be something like `describe("Profile") do` and the features test will contain feature tests that start on the `Profile#Edit` page. This is unclear and hard to parse. Let's improve our feature test titles! Please comment with your recommendations below.

**Comment by Yeison2020** [comment]
Hey ChaelCodes, I hope everything is going well for you. If you need help with anything, I'd be delighted to help. This is my first time contributing to an open-source project.

**Comment by ChaelCodes** [comment]
> Hey ChaelCodes, I hope everything is going well for you. If you need help with anything, I'd be delighted to help. This is my first time contributing to an open-source project. 

Hi @Yeison2020 ,

I'm sorry I didn't get back to you in 2020. You deserved a reply then. I hope you've found your way into open-source since then! 

- Chael
